app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,57 @@
+module SessionsHelper
+
+  def log_in(user)
```

Comment:
`log_in` and `log_in_as` are absolutely the same methods, please keep only one of those two.

Reply:
deleted repeating method